api/auth/authproviders.py

```diff
@@ -66,7 +66,7 @@ def validate_code(self, code, **kwargs):
             'access_token': code,
             'uid': uid,
             'auth_type': self.auth_type,
-            'expires': datetime.datetime.utcnow() + datetime.timedelta(hours=1)
+            'expires': datetime.datetime.utcnow() + datetime.timedelta(days=14)
```

Comment:
Cool. Can you add either a comment talking about why this duration is set, or add a ticket to make the value configurable?